cmd/eth/client.go

```diff
@@ -29,3 +29,20 @@ func client() *seth.Client {
 	os.Exit(1)
 	return nil
 }
+
+func latest(c *seth.Client) int64 {
+	b, err := c.GetBlock(seth.Latest, false)
```

Comment:
It looks like you could parameterize the bool value here and then use `latest` and `pending` functions for the switch statement in the `block` function above.

Reply:
True. I'll send a PR.